src/components/NavBar/NavBar.tsx

```diff
@@ -0,0 +1,30 @@
+import { NavLink } from 'react-router-dom';
+
+export const NavBar = () => {
+  const getLinkClass = ({ isActive }: { isActive: boolean }) => {
+    return isActive
+      ? 'navbar-item is-active has-background-grey-lighter'
+      : 'navbar-item';
```

Comment:
Only one classes change, also to set classes conditionally use [classnames](https://www.npmjs.com/package/classnames) or [clsx](https://www.npmjs.com/package/clsx) library